Drivers/Inc/Contactors.h

```diff
@@ -16,8 +16,26 @@
 #include "BSP_GPIO.h"
 #include "stm32f4xx_gpio.h"
 
+#define CONTACTOR_SENSE_DELAY 1
+
+
+// Masks for Contactor driver board's can message
+#define ARRAY_PRECHARGE_SENSE_FAULT(dataBuf)   ((dataBuf[1] >> 0) & 0x01) // Bit 8: Array precharge sense fault happened
+#define ARRAY_PRECHARGE_EXPECTED_VALUE(dataBuf) ((dataBuf[0] >> 7) & 0x01) // Bit 7: Expected array precharge sense value
```

Comment:
```suggestion
#define ARRAY_PRECHARGE_EXPECTED_VALUE(dataBuf) ((bool)(dataBuf[0] & 0x80)) // Bit 7: Expected array precharge sense value
```
something like this would return a bool

Reply:
don't rly matter just suggestion